Wangscape/AlphaCalculator.cpp

```diff
@@ -0,0 +1,85 @@
+#include "AlphaCalculator.h"
+#include <algorithm>
+#include <numeric>
+
+
+
+AlphaCalculator::AlphaCalculator()
```

Comment:
Agreed. This (and others like it) can be removed.

Visual Studio automatically makes constructors when creating a class; I should look for a way to disable that feature.